pep517/wrappers.py

```diff
@@ -253,7 +253,7 @@ def _call_hook(self, hook_name, kwargs):
             # Run the hook in a subprocess
             with _in_proc_script_path() as script:
                 self._subprocess_runner(
-                    [sys.executable, str(script), hook_name, td],
+                    [sys.executable, abspath(script), hook_name, td],
```

Comment:
This probably needs to be `abspath(str(script))`; `abspath` does not accept a path-like (only str) prior to 3.6.

Reply:
I see, thanks for the feedback!